### ISSUE TITLE
First run of SonarLint on 12/4

### ISSUE THREAD

**Issue by DQueiser** [LeeYang2019/indieProjectDemo#6]
Ran SonarLint against the entire project. Found 175 issues in 33 files. About 117 of them are marked "red", which I believe is more urgent than those marked green.

**Comment by pawaitemadisoncollege** [comment]
@DQueiser Did you complete the peer review feedback form that was in week 14?

**Comment by DQueiser** [comment]
Yes, it was uploaded into Lee's repository. I also added an Issue that had
the results of our SonarLint run.
